projects/components/src/datagrid/datagrid.component.html

```diff
@@ -1,7 +1,10 @@
 <clr-datagrid [clrDgLoading]="isLoading" [ngClass]="this.clrDatagridCssClass" (clrDgRefresh)="gridStateChanged($event)">
+    <!--The clrDgField being passed below is only used for enabling sorting and not filtering. This is because, it was
+        not possible to switch sorting using clrDgColumn.sortBy property-->
     <clr-dg-column
         *ngFor="let column of columnsConfig"
         [clrDgField]="column.queryFieldName"
+        [clrDgSortBy]="column.sortBy"
```

Comment:
This might force the grid to sort locally in addition to pushing the refresh event that causes the server-side filtering. Does using `clrDgSortBy` have precedence in places in VCD-UI?